lib/buyers/urls.py

```diff
@@ -6,6 +6,7 @@
 
 router = DefaultRouter()
 router.register(r'buyer', views.BuyerViewSet)
+router.register(r'method', views.PaymentMethodViewSet, base_name='method')
```

Comment:
I think ``paymethod`` would make this more clear. Since it's attached to the generic API, payments are not necessarily implied.